src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -936,6 +936,17 @@
     <Field type="ThreadState" name="state" label="Thread State" />
   </Event>
 
+  <Event name="CPUTimeSample" category="Java Virtual Machine, Profiling" label="CPU Time Method Profiling Sample" description="Snapshot of a threads state from the CPU time sampler"
```

Comment:
I think we can drop "Profiling" from the label, "CPU Time Method Sample"

Reply:
Will people still understand it?